dashboard/src/components/settings/signInMethods/ConfirmProvidersUpdatedSettings/ConfirmProvidersUpdatedSettings.tsx

```diff
@@ -0,0 +1,90 @@
+import { useDialog } from '@/components/common/DialogProvider';
+import { useCurrentWorkspaceAndApplication } from '@/hooks/useCurrentWorkspaceAndApplication';
+import { Alert } from '@/ui/Alert';
+import Button from '@/ui/v2/Button';
+import ArrowSquareOutIcon from '@/ui/v2/icons/ArrowSquareOutIcon';
+import Link from '@/ui/v2/Link';
+import Text from '@/ui/v2/Text';
+import { toastStyleProps } from '@/utils/settings/settingsConstants';
+import { useConfirmProvidersUpdatedMutation } from '@/utils/__generated__/graphql';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+
+export default function ConfirmProvidersUpdatedSettings() {
+  const { currentApplication } = useCurrentWorkspaceAndApplication();
+  const { openAlertDialog } = useDialog();
+  const [visible, setVisible] = useState(true);
+
+  const [confirmProvidersUpdated] = useConfirmProvidersUpdatedMutation({
+    variables: {
+      id: currentApplication?.id,
+    },
+  });
+
+  async function handleSubmit() {
+    const confirmProvidersUpdatedPromise = confirmProvidersUpdated();
+
+    await toast.promise(
+      confirmProvidersUpdatedPromise,
+      {
+        loading: 'Confirming...',
+        success: 'All done!',
+        error: 'An error occurred while confirming.',
+      },
+      toastStyleProps,
+    );
+
+    setVisible((prev) => !prev);
+  }
+
+  function handleConfirmDialog() {
+    openAlertDialog({
+      title: 'Confirm all providers updated?',
+      payload: (
+        <Text variant="subtitle1" component="span">
+          Please make sure to update all providers before continuing. Your
+          signin flows might break if you don&apos;t.
+        </Text>
+      ),
+      props: {
+        primaryButtonText: 'Confirm',
+        onPrimaryAction: () => handleSubmit(),
+      },
+    });
+  }
+
+  if (!visible) {
+    return null;
+  }
+
+  return (
+    <Alert className="grid items-center grid-flow-col gap-2 p-4 place-content-between bg-amber-500">
```

Comment:
The last one from me 🎨  

```suggestion
    <Alert className="grid items-center place-items-center grid-flow-row lg:grid-flow-col gap-2 p-4 lg:place-content-between bg-amber-500">
```